docker/containers/vitepress/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:18
```

Comment:
To je k čemu?

Reply:
Generuje to pěknější dokumentaci, která jde zobrazit v prohlížeči (readme.localhost). Jede to na jiném nodu a za mě je lepší to nemíchat s aplikací.

Reply:
To je asi na tobě. Mě tam dost dokumentace chybí, ale chápu že není čas a energie ji psát. Dal jsem to tam hlavně proto aby bylo vidět jak to vypadá a funguje. Tohle je za mě nice to have

Reply:
Ok, v tom případě udělám revert změn dokumentace